core/livepeernode_test.go

```diff
@@ -35,6 +35,10 @@ func stubTranscoderWithProfiles(profiles []ffmpeg.VideoProfile) *StubTranscoder
 	return &StubTranscoder{Profiles: profiles}
 }
 
+func (t *StubTranscoder) EndTranscodingSession(sessionId string) {
+
+}
+
```

Comment:
Is it empty function?

Reply:
Yes, it's a stub for testing to support the interface.